de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/StoppableDocumentListener.java

```diff
@@ -50,7 +50,7 @@ public void beforeDocumentChange(DocumentEvent event) {
                 .getFile(document);
 
             if (virtualFile == null) {
-                LOG.debug("Ignoring event for document " + document
+                LOG.trace("Ignoring event for document " + document
```

Comment:
Just for the future, generating the log message even it is not logged requires 3 string concats and a toString method call. I do not think it will haven that often and consumes much CPU time but you should really think about

if (LOG.isTraceEnabled()) … before such log statements